utils/groovy/src/main/java/org/apache/brooklyn/util/groovy/GroovyJavaMethods.java

```diff
@@ -75,6 +76,20 @@
         }
     }
 
+    /**
+     * Alternative implementation of {@link ScriptBytecodeAdapter#isCase(Object, Object)}<br>
+     * Stripped down to work only for caseExpression of type <code>java.lang.Class</code>.<br>
```

Comment:
I think it is the same.
I prefer reusing the actual method being used so you can easily compare implementations of  `ScriptBytecodeAdapter#isCase(Object, Object)` and `safeGroovyIsCase` just by checking that at the end both refer to `DefaultGroovyMethods.isCase`